### ISSUE TITLE
Fragment function showing texture without explicitly using it

### ISSUE THREAD

**Issue by Skyvastern** [godotengine/godot-docs#7497]
**Your Godot version:**
Godot v4.0.3.stable

**Issue description:**
Under the `Using TEXTURE built-in` subheading, for the following example code -
```
void fragment(){
  //this shader will result in an all white rectangle
  COLOR.b = 1.0;
}
```
It says that it will show a white rectangle, without setting a texture. But in Godot, the texture still shows up, and color is added over it. So basically, the above code does the same thing as this -
```
void fragment(){
  COLOR = texture(TEXTURE, UV); //read from texture
  COLOR.b = 1.0; //set blue channel to 1.0
}
```

So I think either something is wrong in the docs, or something is wrong in Godot.

**URL to the documentation page:**
https://docs.godotengine.org/en/stable/tutorials/shaders/your_first_shader/your_first_2d_shader.html#using-texture-built-in

**Comment by clayjohn** [comment]
I think this may be a duplicate of https://github.com/godotengine/godot-docs/issues/7458

**Comment by skyace65** [comment]
This was fixed by #8583.
